pkg/thing/interactors/update_config_test.go

```diff
@@ -130,59 +144,32 @@ var updateConfigTestCases = []UpdateConfigTestCase{
 		&mocks.FakeThingProxy{ReturnErr: errProxyUpdateConfig},
 		&mocks.FakePublisher{},
 	},
-	{
-		"failed to updade thing's config if it doesn't have schema yet",
```

Comment:
Remember these validations still make sense for config updates that contain only an `event` object.

Reply:
Done!